mParticle-OneTrust/MPKitOneTrust.m

```diff
@@ -55,6 +73,64 @@ - (void)start {
     });
 }
 
+// Listener for Consent Change dispatched by One Trust
+-(void)actionConsent:(NSNotification*)notification {
+    NSString *category = notification.name; // Cookie Name
+    NSNumber *status = notification.object; // BOOL-ish value for consent
+    category = notification.name;
```

Comment:
is there a reason you setting it twice?

Reply:
I can't remember why i'm doing this but I'll check.